recipes/mdutils/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "mdutils" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/47/4c/92cf4d088199607d62f59f3c2ce4c0c44fee509d7bfdef5650e5dfe8b363/{{ name }}-{{ version }}.tar.gz
+  sha256: fa17c8aa3391a484fab4046ea55dd99bd516d276d73214c7e411042f648e8a90
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Since this is pure python, it can be a noarch recipe:
```suggestion
  number: 0
  noarch: python
```